flink-filesystems/flink-azure-fs-hadoop/src/test/java/org/apache/flink/fs/azurefs/AzureFileSystemBehaviorITCase.java

```diff
@@ -102,49 +110,48 @@ private static boolean isHttpsTrafficOnly() throws IOException {
                 .enableHttpsTrafficOnly();
     }
 
-    @BeforeClass
-    public static void checkCredentialsAndSetup() throws IOException {
+    @BeforeAll
+    static void checkCredentialsAndSetup() {
         // check whether credentials and container details exist
-        Assume.assumeTrue(
-                "Azure container not configured, skipping test...",
-                !StringUtils.isNullOrWhitespaceOnly(CONTAINER));
-        Assume.assumeTrue(
-                "Azure access key not configured, skipping test...",
-                !StringUtils.isNullOrWhitespaceOnly(ACCESS_KEY));
+        assumeThat(CONTAINER)
+                .describedAs("Azure container not configured, skipping test...")
+                .isNotBlank();
+        assumeThat(ACCESS_KEY)
+                .describedAs("Azure access key not configured, skipping test...")
+                .isNotBlank();
 
         // initialize configuration with valid credentials
         final Configuration conf = new Configuration();
         // fs.azure.account.key.youraccount.blob.core.windows.net = ACCESS_KEY
         conf.setString("fs.azure.account.key." + ACCOUNT + ".blob.core.windows.net", ACCESS_KEY);
```

Comment:
Shouldn't we also add an assume on `ACCOUNT` since we're using it in the parameter name and when generating the base URL? :thinking: